src/Passport.php

```diff
@@ -613,6 +620,36 @@ public static function token()
         return new static::$tokenModel;
     }
 
+    /**
+     * Set the token repository.
+     *
+     * @param \Laravel\Passport\Contracts\TokenRepository|null $tokenRepository
+     * @return \Laravel\Passport\Contracts\TokenRepository|void
+     */
+    public static function useTokenRepository(
+        Contracts\TokenRepository $tokenRepository = null
+    ) {
+        if (is_null($tokenRepository)) {
+            return static::$tokenRepository = new TokenRepository();
+        }
```

Comment:
I would only allow this method to set a token repository and then register the base one in the service provider.